srcpkgs/litterbox/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'litterbox'
+pkgname=litterbox
+version=1.6
+revision=1
+#archs="i686 x86_64"
+build_style=gnu-configure
+configure_args="--prefix=/usr --mandir=/usr/share/man"
```

Comment:
I believe this line is also not needed. It builds fine without it, why did you decide to include it?

Reply:
Since it was throwing warnings with the defaults, if that isn't a problem, I will remove it :sweat_smile:

Reply:
Do you mean these warnings? They are harmless.
```
warning: unsupported option --sysconfdir=/etc
warning: unsupported option --sbindir=/usr/bin
warning: unsupported option --bindir=/usr/bin
warning: unsupported option --infodir=/usr/share/info
warning: unsupported option --localstatedir=/var
warning: unsupported option --host=x86_64-unknown-linux-gnu
warning: unsupported option --build=x86_64-unknown-linux-gnu
```

Reply:
I see, will remove it then :sweat_smile: